app/views/posts/index.html.erb

```diff
@@ -1,6 +1,48 @@
-<h1>Here is the post for the user <%= @posts[:id] %></h1>
-<ul>
-<li>Post: <%= @posts[:post_text] %></li>
-<li>Number of likes: <%= @posts[:likes] %></li>
-<li>Number of comments: <%= @posts[:comments] %></li>
-</ul>
\ No newline at end of file
+<div class="container">
+    <div class="row">
+        <div class="col-md-8">
+            <div class="people-nearby">
+                <div class="nearby-user">
+                  <div class="row">
+                    <div class="col-md-2 col-sm-2">
+                      <img src="<%= @user.photo %>" alt="<%= @user.name %>" class="profile-photo-lg">
+                    </div>
+                  <div class="col-md-7 col-sm-7">
+                    <h5 class="mt-2"><%= @user.name %></h5>
+                  </div>
+                  <div class="col-md-3 col-sm-3">
+                    <p class="text-muted mt-5">Number of posts <%= @user.post_counter %></p>
+                  </div>
+                </div>
+              </div>
+            </div>
+    	</div>
+	</div>
+    <div class="card mb-5" style="width: 100%;">
+      <div class="card-body">
+        <h5 class="card-title">Bio</h5>
+        <p class="card-text"><%= @user.bio %></p>
+      </div>
+    </div>
+    <% @posts.each_with_index do |post, index| %>
+    <div class="card mb-2" style="width: 100%;">
+    <div class="card-body">
+      <h5 class="card-title"><%= link_to "Post #{index + 1}", user_post_path(@user, post)%></h5>
+      <p class="card-text"><%= post.text %></p>
+      <div class="col-md-3 text-md-end col-sm-3" style="width: 100%;">
+        <p class="text-muted mt-5"><span>Likes: <%= post.like_counter %></span> <span>Comments: <%= post.comment_counter %></span></p>
+      </div>
+    </div>
+  </div>
+  <div class="card mb-5" style="width: 100%;">
+  <% post.comments.each do |comment| %>
+    <div class="card-body">
+      <p class="card-text"><%= comment.user.name %>: <%= comment.text %></p>
+    </div>
+    <% end %> 
+  </div>
```

Comment:
- Hi, well done so far. In trying to create at least 6 posts comments on one of the posts, to check if your codes are working, Here is the feedback I am getting. Please check the picture below and fix the bag it is part of the requirements of this milestone.

![Capture](https://github.com/mahdinoori2000/blog-app/assets/63972670/ddb73101-3c8c-4f0d-a1ab-a73efec81678)

Reply:
@OLIPLICHE kindly try to use this code format it will word:

Comment.create(user: user_name, post: post_name, text: 'Super Cool!')


thanks